dev/io.openliberty.jakarta.connectors.2.1/bnd.bnd

```diff
@@ -0,0 +1,32 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.jakarta.connectors.2.1; singleton:=true
```

Comment:
Is this bnd file missing the java 11 bits?
```
javac.source: 11
javac.target: 11

Require-Capability: osgi.ee; filter:="(&(osgi.ee=JavaSE)(version=11))"
```

Reply:
Also, I don't think you need this project at all. This API bundle can be folded into `io.openliberty.jakarta`

Reply:
it is already there - https://github.com/OpenLiberty/open-liberty/blob/integration/dev/io.openliberty.jakarta/connector.2.1.bnd